package.json

```diff
@@ -30,6 +30,7 @@
     "@ant-design/icons": "^4.2.2",
     "@commitlint/cli": "^11.0.0",
     "@commitlint/config-conventional": "^11.0.0",
+    "@googlemaps/google-maps-services-js": "3.1.16",
```

Comment:
This library is not recommended to be used for client-side, but I am only using it for it's interfaces